lib/csp.js

```diff
@@ -19,8 +37,10 @@ module.exports = function (options) {
         name += '-Report-Only';
     }
 
-    for (key in policyRules) {
-        value += key + ' ' + policyRules[key] + '; ';
+    if (typeof policyRules === 'object') {
+        value += parsePolicyRules(policyRules);
+    } else {
+        value += policyRules + '; ';
```

Comment:
maybe this is a part I'm not 100% sure about. It allow to define policy as a string in options and adds final `; `. Maybe this should be confusing and it will be better to just support for arrays and object since this is kind of duplicate to `[''block-all-mixed-content']`. What do you think?